cv/cv/src/cv_utils/camera_projection.py

```diff
@@ -0,0 +1,117 @@
+import json 
+import os 
+
+import cv2
+import numpy as np
+import rospkg
+
+
+class CameraProjection:
+    def __init__(self):
+        rospack = rospkg.RosPack()
+        save_path = rospack.get_path('cv') + '/config/depth_vals.json'
+        if os.path.exists(save_path):
+            print("Using depth map values from", save_path)
+            self.depth_map = json.load(open(save_path, 'r'))
+        else:
+            self.depth_map = self.gen_depth_to_y_map(1.5, 4)
+
+        self.depth_values = []
+        self.keys = {}
+        for i, key in enumerate(self.depth_map):
+            self.keys[key] = i
+            self.depth_values += [self.depth_map[key]]
+        self.depth_values = np.array(self.depth_values)
+
+
+        '''
+        Raw parameters (found in /usr/local/zed/settings)
+        fx = 1400.13
+        fy = 1400.13
+        cx = 1188.06
+        cy = 645.894
+
+        image size: 360, 640, 3
+
+        detination image size: 180, 330
+        ratio: 0.5, 0.515625
+
+        TODO: Remove estimates 
+        '''
+
+        self.fx = (360 + 640) / 2
+        self.fy = (360 + 640) / 2
+        self.ox = 640 / 2
+        self.oy = 360 / 2
```

Comment:
mention a TO DO for getting camera intrinsics from ROS topic. Could be ok for now

Reply:
Guys the TODO is literally already in the file lol